core/input/generisdata_nur_ren.prn

```diff
@@ -4,29 +4,28 @@
 *    The potential is categorized into different greades.
 
 
-
-                wind        spv     hydro    geohdr     geohe       csp   storspv  storwind  storcsp   gridspv  gridwind  gridcsp
-nur.1           0.31      0.230     0.550     1.000     1.000      0.72      1         1        1        0.85      0.85     0.85
-nur.2           0.28      0.215     0.450                          0.68
-nur.3           0.24      0.200     0.350                          0.65
-nur.4           0.20      0.185     0.250                          0.61
-nur.5           0.17      0.172     0.150                          0.57
-nur.6           0.15      0.160                                    0.53
-nur.7           0.13      0.150                                    0.48
-nur.8           0.12      0.140                                    0.43
-nur.9           0.11      0.130                                    0.40
-nur.10          0.10
-nur.11          0.08
-nur.12          0.07
-
-
-maxprod.1         50                    8     0.112      50.0
-maxprod.2         40                   16
-maxprod.3         40                   18
-maxprod.4         40                    8
-maxprod.5         40
-maxprod.6         40
-maxprod.7         60
+              windon   windoff        spv     hydro    geohdr     geohe       csp   storspv  storwindon  storwindoff  storcsp   gridspv  gridwindon  gridwindoff  gridcsp
+nur.1           0.31   0.31         0.230     0.550     1.000     1.000      0.72      1         1            1          1        0.85       0.85       0.85       0.85
```

Comment:
Is this addition of values for windoff needed for the model to run? 

I mean for most technologies the values for nur and maxprod anyway come from f_maxProdGradeRegiXXX, so why add fake values here? 
For the other technologies, this here are remnants from days of old that weren't cleaned up when the maxProdGradeRegi was added. They might be kicked out for clarity's sake, or left in for nostalgia. But I wouldn't add NEW entries here.

Reply:
Noted and removed!

Reply:
Linking separate PR that i still need to test: https://github.com/remindmodel/remind/pull/1837